### ISSUE TITLE
[BUG] Autotitle does not default to current chat model 

### ISSUE THREAD

**Issue by Immortalin** [enricoros/big-AGI#599]
### Description

When both Anthropic and Open AI models are available, Autotile default to the anthropic models for autotitle even when the chat is using the Open AI models. When one account is out of credits, the entire feature fails. It should default to the model used in the active chat.

### Device and browser

Chrome

### Screenshots and more

![image](https://github.com/user-attachments/assets/479ad492-787b-4112-b13e-b43cd475838f)

### Willingness to Contribute

- [ ] 🙋‍♂️ Yes, I would like to contribute a fix.

**Comment by enricoros** [comment]
Good point @Immortalin . Will be fixed for the next release.

**Comment by powellnorma** [comment]
This is this only relevant when one hasn't explicitly configured a model as "fast" model, right?

**Comment by enricoros** [comment]
Correct. The fast model should be the one. But it's too hidden to figure out. I need to have a more prominent selector.

**Comment by enricoros** [comment]
Two things happened here, and only one of them is done. The implicit vendor preference is gone: auto-title runs on the Fast Utility assignment, which you set in Settings > Chat AI or with the Fast chip in any model's options, and which resolves lowest-cost-first when left on Auto. That was the 'too hidden' half, and the selector is now in plain sight. What is not fixed is the half you actually hit: if the model assigned there errors, out of credits or otherwise, auto-title fails silently. There is no fallback to another model and nothing surfaces in the UI. Keeping this open for that, retitled.
